assembl/static2/js/app/components/debate/brightMirror/fictionPreview.jsx

```diff
@@ -100,9 +110,12 @@ const FictionPreview = ({
   const name = authorName || '';
   const author = `${ellipsis(name, FICTION_PREVIEW_NAME_MAX_CHAR)} - `;
```

Comment:
Not possible to manage it in the CSS?

Reply:
@paolina92 ellipsis? How do you do that in CSS ?

Reply:
@cedrtang ```
  width: 250px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
```

Something like this maybe?